src/ralph/backends/http/async_lrs.py

```diff
@@ -49,8 +49,8 @@ class LRSStatementsQuery(BaseQuery):
 
     # pylint: disable=too-many-instance-attributes
 
-    statementId: Optional[str]  # pylint: disable=invalid-name
-    voidedStatementId: Optional[str]  # pylint: disable=invalid-name
+    statement_id: Optional[str] = Field(None, alias="statementId")
```

Comment:
Same here, is None necessary? I already used `Field()` in pydantic for xAPI models and only alias was enough

Reply:
I don't think so. https://stackoverflow.com/a/70993458

Reply:
I was inspired by the Optional Query parameters, inside the API router

Reply:
Actually, migrating to `pydantic v2` requires to specify default `None` even for `Optional`. They removed it for clarity (`Optional` only indicates that a value MAY be `None`). I suggest we use this syntax even with `pydantic V1`, for clarity.